website/content/docs/agent-and-proxy/agent/apiproxy.mdx

```diff
@@ -6,14 +6,24 @@ description: >-
   for Vault's API.
 ---
 
-# Vault agent API proxy
+# Vault Agent API proxy
 
 Vault Agent's API Proxy functionality allows you to use Vault Agent's API as a proxy
```

Comment:
Same here -- stick with "Vault Proxy" or "Vault Agent in Proxy mode"

Reply:
As above, this is referring to Vault Agent's API proxy (i.e. the thing we moved to Vault Proxy but didn't yet remove from Agent), not Proxy's.